### DISCUSSION TITLE
Fan of Star A and SX1

### DISCUSSION THREAD

**Discussion by fooandbar** [fooandbar/Mks-Robin-Nano-Marlin2.0-Firmware-For-Selpic-Star-A.LABISTS-SX1#6] [General]
I disassembled and compared the heads of the SX1 and Star A.
# Hot-end fan
This is Star A's noisy fan.
<img src='https://user-images.githubusercontent.com/63621440/121603618-01d28600-ca84-11eb-93aa-4bf6355bf632.jpg' width=200 />
This is SX1's quiet fan.
<img src='https://user-images.githubusercontent.com/63621440/121603726-2fb7ca80-ca84-11eb-8e74-1a738d48c442.jpg' width=200 />
Different fans are used for SX1 and Star A.
The rated current of the fan used in the SX1 is smaller than that of the Star A, so I think that noise is suppressed by lowering the air volume.
The color of wire is different between SX1 and Star A.
# etc...
<img src='https://user-images.githubusercontent.com/63621440/121603474-c932ac80-ca83-11eb-9c01-bb7549aacea3.png' width=500 />
<img src='https://user-images.githubusercontent.com/63621440/121603516-db144f80-ca83-11eb-9520-6bc0a8b9d82d.png' width=300 />

**Comment by berga** [comment]
I was able to move the HE wires back of the layer-cooling-fan but this wasn't a solution for the noise.
Would the Noctua NF-A4X10 FLX be a good replacement?
Moreover we could redesign the airflow for a vertical fan.

**Reply by fooandbar** [reply]
> Would the Noctua NF-A4X10 FLX be a good replacement?

hi, looks good.

**Reply by berga** [reply]
I'm using the new fan and is very noisless! The temperature grow up fastly and low down slowly but I think the air flow would be right.
